src/wallet/rpc/spend.cpp

```diff
@@ -73,6 +73,38 @@ std::set<int> InterpretSubtractFeeFromOutputInstructions(const UniValue& subtrac
     return set_sffo;
 }
 
+std::vector<CRecipient> ParseOutputs(const UniValue& outputs_in, const UniValue& options)
```

Comment:
Let's not pass all the options as we need only one of them.

Reply:
We are using either `subtract_fee_from_outputs` or `subtractFeeFromOutputs`, depending on which one is present. I think handling that logic in the `ParseOutputs` function is better. Since the options object is passed by reference there isn't any performance difference in passing a reference to the entire options object vs passing a single key.